bokehjs/src/lib/core/properties.ts

```diff
@@ -196,49 +233,49 @@ export class Direction extends Enum<enums.Direction> {
-export class UpdateMode extends Enum<enums.UpdateMode> {}
-export class HTTPMethod extends Enum<enums.HTTPMethod> {}
-export class Logo extends Enum<enums.Logo> {}
-export class RadiusDimension extends Enum<enums.RadiusDimension> {}
-export class Sort extends Enum<enums.Sort> {}
-export class ButtonType extends Enum<enums.ButtonType> {}
-export class HexTileOrientation extends Enum<enums.HexTileOrientation> {}
-export class TapBehavior extends Enum<enums.TapBehavior> {}
-export class BoxOrigin extends Enum<enums.BoxOrigin> {}
-export class HoverMode extends Enum<enums.HoverMode> {}
-export class PointPolicy extends Enum<enums.PointPolicy> {}
-export class LinePolicy extends Enum<enums.LinePolicy> {}
-export class MarkerType extends Enum<enums.MarkerType> {}
+export const AngleUnits = Enum(enums.AngleUnits)
+export const Dimension = Enum(enums.Dimension)
+export const Dimensions = Enum(enums.Dimensions)
+export const FontStyle = Enum(enums.FontStyle)
+export const LatLon = Enum(enums.LatLon)
+export const LineCap = Enum(enums.LineCap)
+export const LineJoin = Enum(enums.LineJoin)
+export const Location = Enum(enums.Location)
+export const LegendClickPolicy = Enum(enums.LegendClickPolicy)
+export const LegendLocation = Enum(enums.LegendLocation)
+export const Anchor = Enum(enums.Anchor)
+export const Orientation = Enum(enums.Orientation)
+export const OutputBackend = Enum(enums.OutputBackend)
+export const RenderLevel = Enum(enums.RenderLevel)
+export const RenderMode = Enum(enums.RenderMode)
+export const Side = Enum(enums.Side)
+export const Place = Enum(enums.Place)
+export const SpatialUnits = Enum(enums.SpatialUnits)
+export const StartEnd = Enum(enums.StartEnd)
+export const VerticalAlign = Enum(enums.VerticalAlign)
+export const TextAlign = Enum(enums.TextAlign)
+export const TextBaseline = Enum(enums.TextBaseline)
+export const TickLabelOrientation = Enum(enums.TickLabelOrientation)
+export const TooltipAttachment = Enum(enums.TooltipAttachment)
+export const Distribution = Enum(enums.Distribution)
+export const StepMode = Enum(enums.StepMode)
+export const SizingMode = Enum(enums.SizingMode)
+export const PaddingUnits = Enum(enums.PaddingUnits)
+export const SliderCallbackPolicy = Enum(enums.SliderCallbackPolicy)
+export const RoundingFunction = Enum(enums.RoundingFunction)
+export const UpdateMode = Enum(enums.UpdateMode)
+export const HTTPMethod = Enum(enums.HTTPMethod)
+export const Logo = Enum(enums.Logo)
+export const RadiusDimension = Enum(enums.RadiusDimension)
+export const Sort = Enum(enums.Sort)
+export const ButtonType = Enum(enums.ButtonType)
+export const HexTileOrientation = Enum(enums.HexTileOrientation)
+export const TapBehavior = Enum(enums.TapBehavior)
+export const BoxOrigin = Enum(enums.BoxOrigin)
+export const HoverMode = Enum(enums.HoverMode)
+export const PointPolicy = Enum(enums.PointPolicy)
+export const LinePolicy = Enum(enums.LinePolicy)
+export const MarkerType = Enum(enums.MarkerType)
```

Comment:
These should all be alphabetized .

Reply:
This maintains order of core/enums, so I updated both. Though, I think, this code will be removed at some point in favor of inline enums.